tools_for_todoist/models/event.py

```diff
@@ -121,15 +126,46 @@ def _last_occurrence(self):
         if not is_allday(start):
             last_occurence = last_occurence.astimezone(start.tzinfo)
         return last_occurence.date() if is_allday(start) else last_occurence
+    
+    def _find_next_occurence(self, rrule_instances):
+        first_exception_start = min(
```

Comment:
split this into multiple lines and statements for clarity

Reply:
also it's not a bad idea to keep track of which exception is the next one. For example if the title of the event was changed in an exception, we should be able to track it.

Reply:
>  For example if the title of the event was changed in an exception, we should be able to track it.

If we're not doing anything with it in this PR, IMO it shouldn't be in the scope of the PR